### ISSUE TITLE
Translation reload strings fail if you don't return array in config

### ISSUE THREAD

**Issue by gutig** [mlocati/concrete5-translate-library#6]
We use the c5 config files in a slightly different way to the default files, so typically they are returning an array as follows;

database.php 

```
<?php

return [
    'default-connection' => 'concrete',
    'connections' => [
        'concrete' => [
            'driver' => 'c5_pdo_mysql',
            'server' => '127.0.0.1',
            'database' => 'concrete5',
            'username' => 'homestead',
            'password' => 'secret',
            'character_set' => 'utf8mb4',
            'collation' => 'utf8mb4_general_ci',
        ],
    ],
];
```

However we use a secrets file and provision that config file with ansible; 

database.php 

```
<?php
$databaseSettings = array(
    'default-connection' => 'concrete',
    'connections' => array(
        'concrete' => array(
            'driver' => 'c5_pdo_mysql',
            'charset' => 'utf8',
        ),
    ),
);

$secrets_file = dirname(__FILE__) . '/secrets.php';
if (file_exists($secrets_file)) include $secrets_file;

return $databaseSettings;
```

And then in secrets.php we do the following; 

```
<?php

// Overrides for /application/config/database.php
$databaseSettings['connections']['concrete']['server'] = '127.0.0.1';
$databaseSettings['connections']['concrete']['database'] = 'ubuntu';
$databaseSettings['connections']['concrete']['username'] = 'ubuntu';
$databaseSettings['connections']['concrete']['password'] = 'ubuntu';

$appSettings['staticAssetVersion'] = '20200113101434';

$appSettings['google']['recaptcha']['site_key'] = 'KEY';
$appSettings['google']['recaptcha']['secret_key'] = 'SECRET';
```

That all works fine, until you reload the strings in the translate site interface. It returns this error as you're trying to find where the array is actually returned and that is no longer there;

`Failed to determine the start of the return array`

Locally I've changed this to do the following in the `src/Util/ConfigFile.php`

```
  public function __construct($filename)
    {
        $this->setArrayFromFile($filename);
    }

    public function setArrayFromFile($filename)
    {
        $this->array = $this->getArrayFromFile($filename);
        return $this;
    }

    public function getArrayFromFile($filename)
    {
        return include $filename;
    }
```

This works fine for me and I was able to reload the strings and then save the file again.  Are we able to amend the way this works?

**Comment by mlocati** [comment]
The problem is that this library should be able to load configuration files for the core and the packages, even if they are not installed (for example, it's able to [extract these strings](https://github.com/concrete5/concrete5/blob/5.7.5.13/web/concrete/config/app.php#L451-L488) by simply providing its path).

IMHO the solution would be to add the possibility to skip a file, maybe by adding a line like this to the configuration file:
```php
/* ccm-translation-library:skip-file */
```

would that work for you?

**Comment by gutig** [comment]
@mlocati yeah I think that would work for me

**Comment by mlocati** [comment]
What about #8?

**Comment by mlocati** [comment]
I just published version 1.6.0 which includes #8. If you are using a composer based concrete5 installation you can simply upgrade the version of the translation library.
